tests/helpers.rs

```diff
@@ -309,13 +320,31 @@ impl TestEnv {
         pic.auto_progress().await;
         info!("pocket-ic server started");
 
+        // fetch the root key of "local" IC and save it to a file
         let root_key = pic.root_key().await.expect("failed to get root key");
         let mut root_key_path = env::current_dir().expect("failed to get working directory");
         root_key_path.push("root_key.der");
         let mut file = File::create(&root_key_path).expect("failed to create file");
         file.write_all(&root_key)
             .expect("failed to write key to file");
 
+        // create a static denylist seed file
+        let denylist_seed = format!(
+            r#"{{
+            "$schema": "./schema.json",
+            "version": "1",
+            "canisters": {{
+                "{}": {{}}
+            }}
+        }}"#,
+            DENYLISTED_CANISTER_ID
+        );
+        let mut denylist_seed_path = env::current_dir().expect("failed to get working directory");
```

Comment:
I think you can just use `fs::write` here (https://doc.rust-lang.org/std/fs/fn.write.html)